snapcraft/internal/pluginhandler.py

```diff
@@ -644,7 +651,7 @@ def _migratable_filesets(fileset, srcdir):
 
 
 def _migrate_files(snap_files, snap_dirs, srcdir, dstdir, missing_ok=False,
-                   follow_symlinks=False):
+                   follow_symlinks=False, fixup_func=None):
```

Comment:
Nah, I take that back--  could see this being useful. I wonder if it's worth supporting multiple fixup_funcs.